test/test_helpers/file_fixtures.rb

```diff
@@ -3,6 +3,6 @@
 ActiveSupport::TestCase.class_eval do
   include ActiveSupport::Testing::FileFixtures
 
-  self.fixture_path = Rails.root.join('test', 'fixtures')
-  self.file_fixture_path = self.fixture_path
+  self.fixture_path = Rails.root.join('test/fixtures')
```

Comment:
```
DEPRECATION WARNING: Passing a path to `fixture_file_upload` relative to `fixture_path` is deprecated.
In Rails 7.0, the path needs to be relative to `file_fixture_path`.

Please modify the call from
`fixture_file_upload("wide.jpg")` to `fixture_file_upload("../../../test/fixtures/wide.jpg")`.
 (called from block (4 levels) in <main> at /home/dmayorov/Projects/3scale/porta/spec/acceptance/api/cms_file_spec.rb:24)

```

Reply:
How this solves that warning? wasn't the path relative before?

Reply:
Sorry, I left the comment on the wrong line :upside_down_face: 

In this file only Rubocop warnings are being fixed:
```
RuboCop: Please use `Rails.root.join('path/to')` instead. [Rails/FilePath]
RuboCop: Redundant `self` detected. [Style/RedundantSelf]
```

The actual fix is adding the line `config.file_fixture_path = config.fixture_path` in `spec/rails_helper.rb` :sweat_smile: (https://github.com/3scale/porta/pull/3847/files#diff-053aa3ea46cb7c6a649e0d0fc592ce729b66e1d44be475516fd58857ba695862R77)

I just checked this file to see why it was not giving warning for integration tests, and decided to fix the warnings along the way.

Reply:
Nice, thanks for clarifying.